packages/serverless-cdk-plugin/src/types/cdkPluginConfig.ts

```diff
@@ -1,5 +1,5 @@
-import { Construct } from 'constructs';
+import { ServerlessConstruct } from './serverlessConstruct';
 
 export type CdkPluginConfig = {
-  cdkConstruct: typeof Construct;
+  serverlessConstruct: typeof ServerlessConstruct;
```

Comment:
we want to keep this as a Construct imo. And dynamically check if it's a serverless Construct to give the props